File: frontend/PersonalMovieList/src/app/movies/movie.ts
export class Movie {
    title: string;
    comment: string;
    rating: number;
    id: number;
    image;

    constructor(id: number, title: string, comment: string, rating: number, image) {
            this.id = id;
            this.title = title;
            this.comment = comment;
            this.rating = rating;
            this.image = image;
        }
}

File: backend/PersonalMovieListApi/Models/Movie.cs
using System.ComponentModel.DataAnnotations;

namespace PersonalMovieListApi.Models
{
    public class Movie 
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string Title { get; set; }
        public string Comment { get; set; }
        [Required]
        public int Rating { get; set; }
    }
}